src/Uno.Extensions.Reactive/Presentation/Bindings/Bindable.cs

```diff
@@ -177,7 +195,7 @@ private void OnOwnerUpdated(T value)
 		// however when we use generated `BindableMyEntity : Bindable<MyEntity>`,
 		// we keep the same instance of `BindableMyEntity` so bindings are not re-evaluated.
 		// In that case we will also raise a 'PropertyChanged("")' in order to force the binding engine to re-evaluate all properties.
-		if (_isInherited)
+		if (_isDenormalizedBindable)
```

Comment:
I'm not sure the usage of the term "normalized" (or denormalized) is appropriate in this context. It's more a mapping than a normalization and we can't really flag it as "Unnormalized form" (https://en.wikipedia.org/wiki/Unnormalized_form)

Reply:
Indeed, will change this in a future commit ... but not sure how to name it :/ Lets discuss about that naming on a chat.